.changeset/gold-shirts-yell.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-org': patch
+---
+
+Support menber list scrollable when parent has specified height
```

Comment:
```suggestion
Support member list scrollable when parent has specified height
```

You also want to add the word "scrollable" to `.github/vale/Vocab/Backstage/accept.txt`